docs/mkdocs/developer/index.md

```diff
@@ -1,18 +1,21 @@
 # Design notes
 
 Kernel Module Management brings a new `Module` CRD.
-`Module` represents an out-of-tree kernel module that must be inserted into the cluster nodes’ kernel through a
-DriverContainer scheduled by a `DaemonSet`.
+
+`Module` represents an out-of-tree kernel module that must be inserted into the cluster nodes’ kernel through a `DriverContainer` scheduled by a `DaemonSet`.
```

Comment:
```suggestion
`Module` represents an out-of-tree kernel module that must be inserted into the cluster nodes’ kernel through a `ModuleLoader` scheduled by a `DaemonSet`.
```

Reply:
Should we also replace all the occurrences of `DriverContainer` in the code to `ModuleLoader`?

Reply:
> Should we also replace all the occurrences of `DriverContainer` in the code to `ModuleLoader`?

Probably would make sense to do it, but my PR was attempting some minor rearranging, I saw that Quentin did a  bigger one with reorg of docs that might include it